spec/requests/user/user_suggestions_spec.rb

```diff
@@ -2,68 +2,106 @@
 
 RSpec.describe "Users", type: :request do
   describe "GET /users" do
-    it "returns no users if state is not present" do
-      get users_path
+    let(:user) { create(:user) }
+    let(:suggested_users_list) { %w[eeyore] }
+    let!(:suggested_user) { create(:user, name: "Eeyore", username: "eeyore", summary: "I am always sad :(") }
 
-      expect(response).to have_http_status(:ok)
-
-      expect(response.parsed_body).to be_empty
+    before do
+      allow(SiteConfig).to receive(:suggested_users).and_return(suggested_users_list)
     end
 
-    it "returns users from the original core team if they are present" do
-      user = create(:user, username: "ben", summary: "Something something")
+    context "when the user is not signed in" do
+      it "returns the default suggested_users from SiteConfig if they are present" do
+        get users_path
+
+        expect(response).to have_http_status(:ok)
 
-      get users_path
+        response_user = response.parsed_body.first
+        expect(response_user["id"]).to eq(suggested_user.id)
+        expect(response_user["name"]).to eq(suggested_user.name)
+        expect(response_user["username"]).to eq(suggested_user.username)
+        expect(response_user["summary"]).to eq(suggested_user.summary)
+        expect(response_user["profile_image_url"]).to eq(ProfileImage.new(suggested_user).get(width: 90))
+        expect(response_user["following"]).to be(false)
```

Comment:
Not a blocker but there's another neat way to [test json with only one expectation ](https://stackoverflow.com/a/8393268)

Reply:
gorgeousssssss, using that now ✨